x/io/int.go

```diff
@@ -11,6 +11,19 @@ var (
 	errValueTooLarge  = errors.New("value too large")
 )
 
+// Used for calculating the number of bytes required to encode a varint.
+const (
+	one   = (1 << 7) >> 1
```

Comment:
nit: Take it or leave it, but you can make this `1 << 6` and same for the rest.

Reply:
I like it like this for readability since varints are based 128 encoded and the reader can clearly see the 7 bits here.